### PR TITLE
Add recipe for minimalkv

### PR DESCRIPTION
**Pull request by xhochy**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/minimalkv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+about:
+  home: https://github.com/data-engineering-collective/minimalkv
+  summary: A key-value storage for binary data, support many backends.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - fjetter
```
Comment (xhochy) [inline recipes/minimalkv/meta.yaml:42]: @fjetter Please 👍 if you are OK with being listed as a maintainer.

File: recipes/minimalkv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - mlondschien
```
Comment (xhochy) [inline recipes/minimalkv/meta.yaml:43]: @mlondschien Please 👍 if you are OK with being listed as a maintainer.

File: recipes/minimalkv/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - siboehm
```
Comment (xhochy) [inline recipes/minimalkv/meta.yaml:44]: @siboehm Please 👍 if you are OK with being listed as a maintainer.

**Review by fjetter — APPROVED** [review]

**Comment by xhochy** [comment]
@conda-forge/staged-recipes This grayskull-generated recipe is ready for review and all maintainers have 👍 .

**Review by dopplershift — APPROVED** [review]

**Pull request merged**